client/src/fluid/FluidShortcuts.res

```diff
@@ -138,6 +138,14 @@ let pNull = (~id=gid(), ()): FluidPattern.t => PNull(id)
 
 let pBlank = (~id=gid(), ()): FluidPattern.t => PBlank(id)
 
+// TUPLETODO use this in tests
```

Comment:
Outstanding todo, FYI

Reply:
Thanks - I'll do this in future PRs, searching throughout for TUPLETODO before marking tuples as no-longer in-progress.